tests/Unit/Domain/Test_Direction.cpp

```diff
@@ -0,0 +1,145 @@
+// Distributed under the MIT License.
```

Comment:
You should look at the codecov report here:
https://codecov.io/gh/sxs-collaboration/spectre/pull/60/diff

and make sure you are covering things :)